gluecodium/src/test/resources/smoke/comments/input/Comments.lime

```diff
@@ -78,9 +78,10 @@ class comments {
     // @return nicely documented
     fun returnCommentOnly(undocumented: String): String
 
-    // Some very useful property.
+    // Status of some property.
     // @get Gets some very useful property.
     // @set Sets some very useful property.
+    // This is description some useful property.
```

Comment:
This line will be invisible for Dart and Swift Platform API and only visible for Java @sets part. Hence, you can duplicate this line and paste after 81 line. 

     // Status of some property.
    // This is description some useful property.
    // @get Gets some very useful property.
    // @set Sets some very useful property.
    // This is description some useful property.

Reply:
Outdated due to new changes.